From 02b187303369d3ce0c19dfb72ffa78f86a3911f0 Mon Sep 17 00:00:00 2001
From: Joseph Sutton <josephsutton@catalyst.net.nz>
Date: Wed, 28 Apr 2021 16:48:55 +1200
Subject: [PATCH] Fix Python docstrings

Signed-off-by: Joseph Sutton <josephsutton@catalyst.net.nz>
Reviewed-by: Andrew Bartlett <abartlet@samba.org>
Reviewed-by: Jeremy Allison <jra@samba.org>

Autobuild-User(master): Jeremy Allison <jra@samba.org>
Autobuild-Date(master): Sat Sep  4 00:55:32 UTC 2021 on sn-devel-184
---
 lib/ldb/pyldb.c                  | 2 +-
 lib/tdb/pytdb.c                  | 2 +-
 lib/tevent/pytevent.c            | 2 +-
 source4/librpc/ndr/py_security.c | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/lib/tevent/pytevent.c b/lib/tevent/pytevent.c
index 93375f71868..62dfe2419ff 100644
--- a/pytevent.c
+++ b/pytevent.c
@@ -573,7 +573,7 @@ static PyMethodDef py_tevent_context_methods[] = {
 	{ "add_timer", (PyCFunction)py_tevent_context_add_timer,
 		METH_VARARGS, "S.add_timer(next_event, handler) -> timer" },
 	{ "add_timer_offset", (PyCFunction)py_tevent_context_add_timer_offset,
-		METH_VARARGS, "S.add_timer(offset_seconds, handler) -> timer" },
+		METH_VARARGS, "S.add_timer_offset(offset_seconds, handler) -> timer" },
 	{ "add_fd", (PyCFunction)py_tevent_context_add_fd, 
 		METH_VARARGS, "S.add_fd(fd, flags, handler) -> fd" },
 	{0},
-- 
GitLab

